metrics/ci_integrations/test/firestore/adapter/firestore_storage_client_adapter_test.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:ci_integration/firestore/deserializer/build_data_deserializer.dart';
+import 'package:ci_integration/firestore/adapter/firestore_storage_client_adapter.dart';
+import 'package:firedart/firedart.dart';
+import 'package:metrics_core/metrics_core.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+import 'package:grpc/grpc.dart';
+
+void main() {
+  group('FirestoreStorageClientAdapter', () {
```

Comment:
Please, consider using double-quotes for the group and all tests descriptions. This style requirement will be in project documents soon.